crates/dbs-virtio-devices/src/lib.rs

```diff
@@ -113,9 +121,25 @@ pub enum Error {
     /// Error from Device activate.
     #[error("Device activate error: {0}")]
     ActivateError(#[from] ActivateError),
+    /// Internal unspecific error.
+    #[error("internal unspecific error")]
+    InternalError,
     /// Error from Interrupt.
     #[error("Interrupt error: {0}")]
     InterruptError(IOError),
+
+    #[cfg(feature = "virtio-net")]
```

Comment:
All virtio-net related Errors are placed in net.rs, and a new NetError is created.

Reply:
done